<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GloveAppClass</class>
 <widget class="QMainWindow" name="GloveAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GloveApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="7">
       <widget class="QSpinBox" name="spinBoxMax14">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>thumb abduction</string>
        </property>
       </widget>
      </item>
      <item row="8" column="7">
       <widget class="QSpinBox" name="spinBoxMax17">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>thumb rotation/TMJ</string>
        </property>
       </widget>
      </item>
      <item row="7" column="7">
       <widget class="QSpinBox" name="spinBoxMax16">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QSpinBox" name="spinBoxMax8">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLineEdit" name="lineEdit15">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QSpinBox" name="spinBoxMax11">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QSpinBox" name="spinBoxMin15">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="label15">
        <property name="text">
         <string>pinkie-ring abduction</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label7">
        <property name="text">
         <string>middle MPJ</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="spinBoxMin6">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="spinBoxMin5">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>thumb MPJ</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinBoxMin4">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QSpinBox" name="spinBoxMax13">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="spinBoxMin10">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label11">
        <property name="text">
         <string>ring PIJ</string>
        </property>
       </widget>
      </item>
      <item row="9" column="5">
       <widget class="QLineEdit" name="lineEdit18">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QSpinBox" name="spinBoxMin9">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label13">
        <property name="text">
         <string>pinkie MPJ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBoxMin2">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBoxMax2">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QSpinBox" name="spinBoxMin11">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="spinBoxMax5">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="QSpinBox" name="spinBoxMax15">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBoxMax1">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEdit11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QSpinBox" name="spinBoxMax12">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QSpinBox" name="spinBoxMin13">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="spinBoxMax4">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label8">
        <property name="text">
         <string>middle PIJ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QSpinBox" name="spinBoxMin16">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label9">
        <property name="text">
         <string>middle-index abd'n</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label10">
        <property name="text">
         <string>ring MPJ</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="lineEdit13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="spinBoxMax7">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxGlove">
          <property name="text">
           <string>手套</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxHand">
          <property name="text">
           <string>机械手</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="4">
       <widget class="QLabel" name="label17">
        <property name="text">
         <string>wrist pitch</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="lineEdit12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="spinBoxMax6">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="9" column="6">
       <widget class="QSpinBox" name="spinBoxMin18">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <widget class="QSpinBox" name="spinBoxMin17">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label12">
        <property name="text">
         <string>ring-middle abduction</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>index MPJ</string>
        </property>
       </widget>
      </item>
      <item row="9" column="7">
       <widget class="QSpinBox" name="spinBoxMax18">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QLabel" name="label16">
        <property name="text">
         <string>palm arch</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QSpinBox" name="spinBoxMin8">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QLineEdit" name="lineEdit16">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>thumb IJ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="spinBoxMax3">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label6">
        <property name="text">
         <string>index DIJ</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QLineEdit" name="lineEdit17">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QSpinBox" name="spinBoxMin12">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEdit" name="lineEdit14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSpinBox" name="spinBoxMax10">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinBoxMin3">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QSpinBox" name="spinBoxMax9">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QSpinBox" name="spinBoxMin14">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="spinBoxMin7">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="label14">
        <property name="text">
         <string>pinkie PIJ</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBoxMin1">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QLabel" name="label18">
        <property name="text">
         <string>wrist yaw</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GloveApp.qrc"/>
 </resources>
 <connections/>
</ui>
